lib/studies/shrine/app.dart

```diff
@@ -164,7 +164,7 @@ class _ShrineAppState extends State<ShrineApp>
           initialRoute: ShrineApp.loginRoute,
           routes: {
             ShrineApp.loginRoute: (context) => const LoginPage(),
-            ShrineApp.homeRoute: (context) => home,
+            ShrineApp.homeRoute: (context) => SelectionArea(child: home),
```

Comment:
is there an `ApplyTextOptions` missing here? if not, can it be moved here for consistency?